src/mount/misc.rs

```diff
@@ -0,0 +1,24 @@
+//! Miscellaneous mount APIs
+
+use crate::backend::mount::types::MountAttr;
+use crate::fd::BorrowedFd;
+use crate::fs::AtFlags;
+use crate::{backend, io, path};
+
+/// `mount_setattr(dir_fd, path, flags, mount_attr)`
```

Comment:
Since there is no `mount_setattr` declaration exposed in C, could you change this comment to `syscall(SYS_mount_setattr, ...)`?

Reply:
glibc added `mount_setattr` in 2.36 ([commit](https://sourceware.org/git/?p=glibc.git;a=commitdiff;h=af1aa36c617d72e80ff6b268bf927b6642d90823)), though the `libc` crate has no support for it.

In my system, the declaration is visible in `/usr/include/x86_64-linux-gnu/sys/mount.h`

```c
/* Change the mount properties of the mount or an entire mount tree.  If
   PATH is a relative pathname, then it is interpreted relative to the
   directory referred to by the file descriptor dirfd.  Otherwise if DFD is
   the special value AT_FDCWD then PATH is interpreted relative to the current
   working directory of the calling process.  */
extern int mount_setattr (int __dfd, const char *__path, unsigned int __flags,
                          struct mount_attr *__uattr, size_t __usize)
  __THROW;
```

Reply:
Ok, cool. As long as it's in glibc, for the purposes of the documentation comment here, that's good enough.